aspnetcore/tutorials/signalr.md

```diff
@@ -74,33 +72,32 @@ At the end, you'll have a working chat app:
 
 * From the menu, select **File > New Solution**.
 
-* Select **.NET Core > App > ASP.NET Core Web App** (Don't select **ASP.NET Core Web App (MVC)**).
+* Select **.NET Core > App > Web Application** (Don't select **Web Application (MVC)**).
 
-* Select **Next**.
+* Make sure the **Target Framework** is set to **.NET Core 3.0**, and then select **Next**.
 
 * Name the project *SignalRChat*, and then select **Create**.
 
 ---
 
 ## Add the SignalR client library
 
-The SignalR server library is included in the `Microsoft.AspNetCore.App` metapackage. The JavaScript client library isn't automatically included in the project. For this tutorial, you use Library Manager (LibMan) to get the client library from *unpkg*. unpkg is a content delivery network (CDN)) that can deliver anything found in npm, the Node.js package manager.
+The SignalR server library is included in the ASP.NET 3.0 shared framework. The JavaScript client library isn't automatically included in the project. For this tutorial, you use Library Manager (LibMan) to get the client library from *unpkg*. unpkg is a content delivery network (CDN)) that can deliver anything found in npm, the Node.js package manager.
 
 # [Visual Studio](#tab/visual-studio/)
 
 * In **Solution Explorer**, right-click the project, and select **Add** > **Client-Side Library**.
 
 * In the **Add Client-Side Library** dialog, for **Provider** select **unpkg**.
 
-* For **Library**, enter `@aspnet/signalr@1`, and select the latest version that isn't preview.
-
-  ![Add Client-Side Library dialog - select library](signalr/_static/libman1.png)
+* For **Library**, enter `@aspnet/signalr@next`.
+<!-- when 3.0 is released, change @next to @latest -->
```

Comment:
Do we have an issue for this change too?

Reply:
Might not need a separate issue, as I may delay merging until after the name change.  Wade found an issue with VS Mac that I need to follow up on.